frontend/pages/model/[modelId].tsx

```diff
@@ -51,15 +52,19 @@ export default function Model() {
             {
               title: 'Registry',
               path: 'registry',
-              view: <ModelImages model={model} />,
+              view: <ModelImages model={model} currentUserRoles={currentUserRoles} />,
             },
             {
               title: 'Inferencing',
               path: 'inferencing',
               view: <InferenceServices model={model} />,
               hidden: !uiConfig.inference.enabled,
             },
-            { title: 'Settings', path: 'settings', view: <Settings entry={model} /> },
+            {
+              title: 'Settings',
+              path: 'settings',
+              view: <Settings entry={model} currentUserRoles={currentUserRoles} />,
+            },
```

Comment:
Not sure how I feel about this. In my opinion the settings tab itself should be restricted?

Reply:
I asked the same thing, but it's important for any user to be able to see the collaborators in the entry access tab. We could simplify a lot of the settings roles logic if we find another place to display that information. I'll make a ticket to explore this idea.